code/game/objects/items/defib.dm

```diff
@@ -483,12 +483,14 @@
 	busy = TRUE
 	M.visible_message("<span class='danger'>[user] has touched [M] with [src]!</span>", \
 			"<span class='userdanger'>[user] has touched [M] with [src]!</span>")
-	M.adjustStaminaLoss(50)
-	M.Paralyze(100)
-	M.updatehealth() //forces health update before next life tick //isn't this done by adjustStaminaLoss anyway?
+	M.adjustStaminaLoss(60)
+	M.Knockdown(75)
+	M.Jitter(50)
+	M.convulsing(150) //makes your hands spaz out and drop items every two lifes or so
```

Comment:
Note that a tick of Life() takes place every TWO seconds, not every second, so convulsing isn't as strong of a status effect as you might think that it is.

Reply:
you're underestimating being almost unable to use items